packages/components/src/components/mobile-dialog/mobile-dialog.tsx

```diff
@@ -31,14 +45,14 @@ const MobileDialog = props => {
     const checkVisibility = () => {
         if (props.visible) {
             document.body.style.overflow = 'hidden';
-            document.getElementById(portal_element_id).style.overflow = 'hidden';
+            (document.getElementById(portal_element_id) as HTMLDivElement).style.overflow = 'hidden';
         } else {
-            document.body.style.overflow = null;
-            document.getElementById(portal_element_id).style.overflow = null;
+            document.body.style.overflow = 'null';
```

Comment:
```suggestion
            document.body.style.overflow = 'unset';
```
@likhith-deriv Shouldn’t be `unset` instead of string `null`? 🤔

Reply:
Looked odd to me too. Since null was give, I just made it as string